pkg/queue/manager.go

```diff
@@ -300,13 +300,14 @@ func (m *Manager) RequeueWorkload(ctx context.Context, info *workload.Info, imme
 		return false
```

Comment:
actually, a new version might be on a different queue (because the user might have changed it).

let's do the get before obtaining the queue from `m.queues`.

Reply:
good catch.